pkgs/servers/sip/freeswitch/default.nix

```diff
@@ -2,9 +2,14 @@
 , openssl, perl, sqlite, libjpeg, speex, pcre
 , ldns, libedit, yasm, which, lua, libopus, libsndfile
 
+, postgresql
+, enablePostgres ? true
+
 , SystemConfiguration
 }:
 
+with stdenv.lib;
```

Comment:
Note that you use `with stdenv.lib;` twice.

You may add `lib` as derivation input (next to `stdenv`), and refer as `lib.optionals` in all other places. This is probably better alternative than just remove second duplicate.